packages/user-operation-controller/package.json

```diff
@@ -74,7 +74,7 @@
     "@metamask/gas-fee-controller": "^17.0.0",
     "@metamask/keyring-controller": "^17.0.0",
     "@metamask/network-controller": "^19.0.0",
-    "@metamask/transaction-controller": "^33.0.0"
+    "@metamask/transaction-controller": "^33.1.0"
```

Comment:
I think we can revert this one, it may have been bumped by accident:

``` suggestion
    "@metamask/transaction-controller": "^33.0.0"

Reply:
Yes, the linter would complain without it

Reply:
Similar point as above — if we can revert this line, that would be ideal, as otherwise it would force a new major version for this package.